### DISCUSSION TITLE
Adding ignore to prettierrc?

### DISCUSSION THREAD

**Discussion by joezappie** [prettier/prettier#13681] [General]
Is it possible to add the ignore list to the prettierrc file? Eslint will let you add them with the ignorePatterns key:

```
 "ignorePatterns": [
    "lib/",
    "data/",
    "build/",
    "config/",
    "jsdoc/"
  ]
```

Would be nice to combine these files as theyre both short and theres so many config files nowadays.

**Comment by kachkaev** [comment]
Yeah that’d be cool. See discussion in https://github.com/prettier/prettier/pull/12672#issuecomment-1256947760

**Comment by Standard8** [comment]
Just to note in case anyone finds this, there's already a couple of issues covering this, e.g. #3460, and #15481. There's also a PR #16286 that I think is the most recent attempt, but unfortunately hasn't received feedback from the maintainers yet.
